src/test/unit/profile-upgrading.test.js

```diff
@@ -50,7 +50,8 @@ describe('upgrading old cleopatra profiles', function() {
     const { markers } = thread;
 
     const markerWithCauseIndex = markers.data.findIndex(
-      marker => marker !== null && marker.type === 'tracing' && marker.cause
+      marker =>
+        marker !== null && marker.type === 'tracing' && 'cause' in marker
```

Comment:
This was because of my changes to the marker payload definitions. Flow allows checking for properties existence through the `in` operator, but not the `.` operator.

```
Cannot call await with unserializeProfileOfArbitraryFormat(...) bound to p because:
 • Either cannot get marker.cause because property cause is missing in NavigationMarkerPayload [1].
 • Or cannot get profile.threads because property threads is missing in Promise [2].

     src/test/unit/profile-upgrading.test.js
       42│   // Executing this only for oldCleopatraProfile because
       43│   // ancientCleopatraProfile doesn't have any causes for markers.
       44│   it('should be able to convert causes from old cleopatra profiles', async function() {
       45│     const profile = await unserializeProfileOfArbitraryFormat(
       46│       oldCleopatraProfile
       47│     );
       48│
       49│     const [thread] = profile.threads;
       50│     const { markers } = thread;

     src/profile-logic/process-profile.js
 [2] 1139│ ): Promise<Profile> {

     src/types/profile.js
 [1]  120│   data: MarkerPayload[],
```